docs/source/markdown/podman-volume-prune.1.md

```diff
@@ -21,14 +21,9 @@ Do not prompt for confirmation.
 
 #### **--filter**
 
-Filter volumes to be pruned. Volumes can be filtered by the following attributes:
+Filter volumes to be pruned. Volumes can be filtered by the following attribute:
```

Comment:
Is this true? I seem to be able to use every one of those (the removed entries) as an argument to `podman volume prune --filter`.

Reply:
You are right. This was apparently only removed from the compat api endpoint.